api_app/analyzers_manager/migrations/0121_analyzer_config_guarddogobservable_go.py

```diff
@@ -0,0 +1,152 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "guarddog_observable.GuarddogObservable",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "name": "GuarddogObservable_go",
+    "description": "GuardDog is a tool that allows to identify malicious PyPI and npm packages or Go modules. It runs a set of heuristics on the package source code (through Semgrep rules) and on the package metadata.\r\n\r\ngo modules observable. Just give the name of the package",
```

Comment:
It's enough a single migration for all the 3 cases (npm,pypi, org) and you can call it `GuarddogObservable` generically

Reply:
you can remove the other 2 and the merge migration